pkg/gatewayserver/gatewayserver_test.go

```diff
@@ -392,6 +405,127 @@ func TestGatewayServer(t *testing.T) {
 				}
 			},
 		},
+		{
+			Protocol:                "basicstation",
+			GatewayStatusMsgSupport: false,
+			ValidAuth: func(ctx context.Context, ids ttnpb.GatewayIdentifiers, key string) bool {
+				// TODO: Add basic Auth header check
```

Comment:
Are we closing this here or are we making a TODO issue?

Reply:
There's already a TODO item in the parent (umbrella) issue and I've added the link: https://github.com/TheThingsNetwork/lorawan-stack/issues/74#issue-404431141

Reply:
Great, I was referring to a missing link indeed